src/extension.ts

```diff
@@ -109,7 +128,7 @@ function executeAmebaOnWorkspace(ameba: Ameba | null) {
     if (!ameba) return;
```

Comment:
```suggestion
    if (!ameba || ameba.config.trigger === LintTrigger.None) return;
```

Reply:
LintTrigger == None doesnt mean lints are never run, just that they aren't run manually. This check should be moved up to the top level on line 85 where this method is called.